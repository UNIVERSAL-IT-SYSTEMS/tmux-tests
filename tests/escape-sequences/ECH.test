#!/bin/sh

S=$(mktemp)
trap 'rm -f $S' 0 1 15

cat <<EOF >$S
printf "\033[Xa"
printf "\nb\010\033[0Xc"
printf "\nd\010\033[1Xe"
printf "\nfgh\010\010\033[2Xi"
printf "\n\033[75Gxxxxxxx\033M\033[76G\033[10X"

$TMUX_L wait -S tests
$TMUX_L wait never
EOF

$TMUX_L new -d -x 80 -y 25 -- sh $S
$TMUX_L wait tests\; capturep -ep -S- -E-
