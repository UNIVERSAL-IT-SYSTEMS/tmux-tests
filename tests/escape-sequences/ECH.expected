a
c
e
fi
                                                                          x
x


















